app/views/oauth_clients/index.html.erb

```diff
@@ -34,10 +34,14 @@
 <p><%= t(".no_apps_html", :oauth => link_to(t(".oauth"), "https://oauth.net")) %></p>
 <% else %>
 <p><%= t ".registered_apps" %></p>
-<% @client_applications.each do |client| %>
-  <div class="client_application">
-    <%= link_to client.name, :action => :show, :id => client.id %>
-  </div>
-<% end %>
+<p>
+  <% @client_applications.each do |client| %>
+    <div class="client_application">
+      <%= link_to client.name, :action => :show, :id => client.id %>
+    </div>
+  <% end %>
+</p>
 <% end %>
-<h3><%= link_to t(".register_new"), :action => :new %></h3>
+<p>
```

Comment:
No need to wrap a button in a paragraph tag (and it's not really correct anyway, since the contents aren't a paragraph of text). 

The final button on the page doesn't need any bottom padding anyway.

Reply:
I was copying off the oauth2 page (another dubious use of <p&gt; there), but yeah it didn't solve my spacing issue anyway. I'll get rid of it again.